tests/unit/users/usersService.test.ts

```diff
@@ -321,4 +321,33 @@ describe('Users Service', () => {
 
     expect(result.minimumLength).toBeDefined();
   });
+
+  it('Should allow a user with permissions to view the email template configurations', async () => {
+    // Response data will be camel cased by the interceptor
+    const templates = {
+      activationEmailTemplateId: randomHexString(24),
+      reactivationEmailTemplateId: randomHexString(24),
+      passwordResetEmailTemplateId: randomHexString(24),
+      oidcUnlinkEmailTemplateId: randomHexString(24),
+    };
+
+    nock(`${host}${USER_BASE}`).get(`/email_templates`).reply(200, templates);
+
+    const result = await sdk.users.getEmailTemplates();
+    expect(result).toEqual(templates);
+  });
+
+  it('Should allow a user with permissions to set the email template configurations', async () => {
+    // The user service will snake case the body
+    const templates = {
+      activationEmailTemplateId: randomHexString(24),
+      reactivationEmailTemplateId: randomHexString(24),
+      oidcUnlinkEmailTemplateId: randomHexString(24),
+    };
+
+    nock(`${host}${USER_BASE}`).put(`/email_templates`).reply(200, templates);
```

Comment:
Is it correct that the request body is missing here? Also, I think we expect the user service the user service to receive and respond with the property names in snake case right?

Reply:
Pogchamp!